### ISSUE TITLE
BuildStream.doap is incorrectly included in MANIFEST.in

### ISSUE THREAD

**Issue by Cynical-Optimist** [buildstream-migration/bst-staging#430]
[See original issue on GitLab](https://gitlab.com/BuildStream/buildstream/-/issues/430)
In GitLab by [[Gitlab user @phildawson]](https://gitlab.com/phildawson) on Jun 19, 2018, 09:40

## Background
BuildStream.doap is incorrectly included in MANIFEST.in

This is a simple typo in MANIFEST.in. And can be easily rectified by replacing `include buildstream.doap` with `include BuildStream.doap`

This was discovered while running `setup.py test`. Just before the test session is started. A warning message is displayed:
```
reading manifest template 'MANIFEST.in'
warning: no files found matching 'buildstream.doap' 
```

## Task description

* Fix typo in MANIFEST.in

## Acceptance Criteria
* [x] BuildStream.doap correctly included in MANIFEST.in. This will prevent the above warning message from being shown.

**Comment by Cynical-Optimist** [comment]
In GitLab by [[Gitlab user @phildawson]](https://gitlab.com/phildawson) on Jun 19, 2018, 09:53

mentioned in commit 40ed78120d4c30f2210aeb6cc4eb9e1da0928c26

**Comment by Cynical-Optimist** [comment]
In GitLab by [[Gitlab user @phildawson]](https://gitlab.com/phildawson) on Jun 19, 2018, 17:48

created branch [`430-buildstream-doap-is-incorrectly-included-in-manifest-in`](https://gitlab.com/BuildStream/buildstream/compare/master...430-buildstream-doap-is-incorrectly-included-in-manifest-in)

**Comment by Cynical-Optimist** [comment]
In GitLab by [[Gitlab user @phildawson]](https://gitlab.com/phildawson) on Jun 19, 2018, 17:48

mentioned in merge request !508

**Comment by Cynical-Optimist** [comment]
In GitLab by [[Gitlab user @phildawson]](https://gitlab.com/phildawson) on Jun 20, 2018, 09:44

marked the task **BuildStream.doap correctly included in MANIFEST.in. This will prevent the above warning message from being shown.** as completed

**Comment by Cynical-Optimist** [comment]
In GitLab by [[Gitlab user @tristanvb]](https://gitlab.com/tristanvb) on Jun 22, 2018, 00:32

mentioned in commit 49fb3636c658936c270548557d3d622a78bf1c51

**Comment by Cynical-Optimist** [comment]
In GitLab by [[Gitlab user @tristanvb]](https://gitlab.com/tristanvb) on Jun 22, 2018, 16:14

closed via merge request !508

**Comment by Cynical-Optimist** [comment]
In GitLab by [[Gitlab user @tristanvb]](https://gitlab.com/tristanvb) on Jun 22, 2018, 16:14

closed via commit 49fb3636c658936c270548557d3d622a78bf1c51
